src/common/filter/FunctionManager.h

```diff
@@ -60,7 +60,6 @@ class FunctionManager final {
         Function                body_;
     };
 
-    mutable folly::RWSpinLock                           lock_;
     std::unordered_map<std::string, FunctionAttributes> functions_;
```

Comment:
I think you better mark it const.

Reply:
This may not help due to ref counter use atomic operation.